packages/dbml-core/src/parse/mssql/base_utils.js

```diff
@@ -0,0 +1,21 @@
+const P = require('parsimmon');
+const wss = require('./whitespaces');
+
+exports.word = function (string) {
+  return P.string(string).skip(wss).desc(`"${string}"`);
+};
+
+function replaceWhitespaceWithRegexp (regexp) {
+  let string = String(regexp);
+  string = string.replace(/[\s]+/g, '[^\\S\\r]+');
```

Comment:
Suggest changing this line to this:
```
string = string.replace(/[\s]+/g, '\\s+');
``` 

FYI, `\s` matches white space, including space, tab, line feed, ...
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions/Character_Classes